thesis-api/main/services/category_service.py

```diff
@@ -14,5 +15,14 @@ def __init__(self, dao: CategoryDAO):
     def build(db: Session):
         return CategoryService(CategoryDAO(db))
 
+    def create_all(self, fn: str):
+        with open(fn, mode='r') as csv_file:
+            csv_reader = csv.DictReader(csv_file, delimiter=';')
+            for row in csv_reader:
+                if self.__dao.exists_with_name(row['name']):
+                    continue
```

Comment:
wspominałaś chyba coś o edytowaniu kategorii w bazie, może zamiast continue lepiej byłoby uaktualnić pole search_word?

Reply:
Właśnie ostatecznie chyba stanęło na tym, że na razie najwyżej drobne zmiany takie będziemy robić ręcznie chyba
No ale mogę to zrobić
Tylko może już puścić ten PR a potem to dorobię